i18n.php

```diff
@@ -2,6 +2,7 @@
     'en'    => array(
         'snippet_context_choice'  => '<snippet_undefined><snippet_keyword>%count%</snippet_keyword> suite has undefined steps. Please choose the context to generate snippets:</snippet_undefined>',
         'snippet_proposal_title'  => '<snippet_undefined><snippet_keyword>%count%</snippet_keyword> has missing steps. Define them with these snippets:</snippet_undefined>',
+        'snippet_proposal_use'    => '{1} Don\'t forget this use statement:|]1,Inf]Don\'t forget these %count% use statements:',
```

Comment:
I don't think the singular is really needed. We will always be printing at least 2 use statements, the one for the current step and the PendingException

Reply:
Correct!